cli/smartcontract/smart_contract.go

```diff
@@ -214,16 +218,42 @@ func NewCommands() []cli.Command {
     * 'string\:string' is a string with a value of 'string:string'
     * '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c' is a
       key with a value of '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c'
+
+   Cosigners represent a set of Uint160 hashes with witness scopes and are used
+   to verify hashes in System.Runtime.CheckWitness syscall. To specify cosigners
+   use [cosigner]:[scope] syntax were
+    * 'cosigner' is hex-encoded 160 bit (20 byte) LE value of cosigner's address,
+                 which could have '0x' preffix.
+    * 'scope' is a set of cosigner's scopes, which could be:
```

Comment:
`... is a comma-separated set of...`

Reply:
Actually, commas are optional

Reply:
And it's a bug because at the moment I can do `hash:notaglobal` and it would be perfectly valid with current code. Also, I don't see `cannot be combined with other flags` enforced in any way.

Reply:
Fixed, now it's really comma-separated and scope string should match precisely with it's lower-cased name.